tensorflow/lite/micro/kernels/fully_connected_test.cc

```diff
@@ -316,16 +316,16 @@ TfLiteStatus TestFullyConnectedFloat(
 }
 #endif
 
-template <typename T>
+template <typename T, typename biasT>
```

Comment:
can we use <typename dataT, typename weightT, typename biasT>? Basically, instead of int8_t* wieghts_quantized below, use weightT weights_quantized to preserve generality?

Reply:
Done